app/src/app/components/specmath-modal/modal.component.html

```diff
@@ -0,0 +1,105 @@
+<!-- Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this spec except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. -->
+
+<div class="modal-container">
+  <mat-dialog-content>
+    <div class="modal-step-header-container">
+      <div class="modal-step-header-labels">
+        <h3>
+          Merge specs <b class="step-indicator">{{ currentStep }}/4</b>
```

Comment:
you can use `{{ currentStep }}/{{ maxSteps }}` now

Reply:
done 💯